src/fontra/views/editor/editor.js

```diff
@@ -1013,6 +1022,34 @@ export class EditorController {
     });
   }
 
+  async doubleClickedGuidelinesCallback(event) {
+    const glyphController = await this.sceneModel.getSelectedStaticGlyphController();
+    const instance = glyphController.instance;
+
+    const guidelineIndex = this.sceneController.doubleClickedGuidelineIndices[0];
+    let guideline = instance.guidelines[guidelineIndex];
+    const { guideline: newGuideline } = await this.doAddEditGuidelineDialog(guideline);
```

Comment:
When we're not "at a source", we should get the "not at a source" dialog instead. We currently only get it _after_ dismissing this dialog. Check the .canEdit attribute of the glyph controller.

Reply:
Thanks a lot for you solid review and testing. It missed in for anchors as well. I fixed it for both.